homeassistant/components/squeezebox/config_flow.py

```diff
@@ -106,6 +110,7 @@ async def _validate_input(self, data):
             data[CONF_PORT],
             data.get(CONF_USERNAME),
             data.get(CONF_PASSWORD),
+            https=data.get(CONF_HTTPS),
```

Comment:
```suggestion
            https=data[CONF_HTTPS],
```

As you specified a default value, the entry is always there.